editors/vscode/syntaxes/.gitignore

```diff
@@ -0,0 +1 @@
+swarm.tmLanguage.json
```

Comment:
Should this say `swarm.tmLanguage.yaml`?  It looks like you deleted the `.json` file.

Reply:
The JSON needs to be generated and you do not want to commit it ;)

Reply:
Ah, now I get it!